src/common/NetworkManager.ts

```diff
@@ -459,7 +459,7 @@ export class NetworkManager extends EventEmitter {
       }
     }
 
-    this._emitRedirectResponse(lastRequest, event.redirectResponse, null);
+    this._emitRedirectResponse(lastRequest, event.redirectResponse, extraInfo);
```

Comment:
I think I have tried by I think it din't make a difference: there is still an extra unprocessed extra info remaining (the one with status code 200)

Reply:
I hope it helps though.